C21_VisionAndLidar/C21_PANORAMA req
---
sensor_msgs/Image res